src/main/java/KafkaTwitter.java

```diff
@@ -0,0 +1,111 @@
+import com.fasterxml.jackson.databind.ObjectMapper;
```

Comment:
https://github.com/saurzcode/twitter-stream/blob/master/src/main/java/com/saurzcode/twitter/TwitterKafkaProducer.java ?

Reply:
I updated the producer to keep reading messages and don't stop at 1000 :)